### PR TITLE
Removed legacy coffeescript, added Gemfile for dependencies, updated compile scripts to use emerald-lang gem.

### PR DESCRIPTION
**Pull request by armcburney**

### CODE REVIEW

File: coffee/parser.coffee [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
-  # Get the emerald code on the page
-  code = $ '#main-code'
-  text = code[0].innerText
-
-  # Regex for keywords - (this is bad I know)
-  keywords = new RegExp 'scripts|styles|metas|each|as', "g"
-  tags = new RegExp 'html|section|main', "g"
-  arrows = new RegExp '->|=>|~>', "g"
-
-  # Parse ard replace keywords - (also bad, I know)
```
Comment (armcburney) [inline coffee/parser.coffee:20]: :joy:

File: coffee/parser.coffee [hunk 2]
```diff
@@ -1,29 +0,0 @@
...
-# the semantic knowledge willl come in when we have the atom
-# syntax highlighter, and we use the highlighting package.
-#
-
-$ ->
-  # Get the emerald code on the page
-  code = $ '#main-code'
-  text = code[0].innerText
-
-  # Regex for keywords - (this is bad I know)
```
Comment (armcburney) [inline coffee/parser.coffee:15]: 👎 😂

**Review by davepagurek — APPROVED** [review]
☕️😂

**Comment by armcburney** [comment]
:joy: :100:

**Pull request merged**